Input/Input.module.less

```diff
@@ -10,6 +10,7 @@
 		box-shadow: 0 0 0 @agate-input-decorator-border-width @agate-input-decorator-border-color;
 		box-sizing: border-box;
 		vertical-align: middle;
+		font-family: @agate-base-font-family;
```

Comment:
Please only use component-based variables in module LESS files. You can assign the relationships in the variables and colors files for the skin. Referring to the general LESS values here effectively hard-codes every skin so that it _must_ use this value, rather than its own preferences.

Reply:
Good to know :)